templates/todo/projects/csharp-cosmos-sql/.repo/bicep/infra/main.parameters.json

```diff
@@ -13,6 +13,9 @@
     },
     "useAPIM": {
       "value": "${USE_APIM=false}"
+    },
+    "apimSku": {
```

Comment:
Why are we choosing to expose `APIM_SKU` at the main module vs just have a default in the APIM module to this value?

Reply:
`APIM_SKU` is needed because the `Consumption` SKU is not supported in US or China clouds 

https://learn.microsoft.com/en-us/azure/api-management/api-management-features

> The Consumption tier isn't available in the US Government cloud or the Microsoft Azure operated by 21Vianet cloud.


This makes it possible to use APIM in different clouds with a safe default for public clouds.